src/lib/markdown/operationLink.ts

```
@@ -0,0 +1,171 @@
+    const operation = openapi.getOperation?.(operationId)
+    if (!operation) {
+      return defaultRender(tokens, idx, options, env, self)
+    }
+
+    const method = openapi.getOperationMethod?.(operationId) || 'get'
+    const title = getOperationTitle(tokens, idx, operation)
+
+    let transformedHref = href
+    if (transformHref) {
+      transformedHref = transformHref(href)
+      tokens[idx].attrs![hrefIndex][1] = transformedHref
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)